scripts/plotting/plot_simple_vs_streaming_shuffle.py

```diff
@@ -0,0 +1,156 @@
+import json
```

Comment:
I already merged this in as `plot_streaming.py`, you can remove this file from the PR.